scripts/embargo_registrations.py

```diff
@@ -46,7 +46,8 @@ def main(dry_run=True):
                         parent_registration.registered_from.add_log(
                             action=NodeLog.EMBARGO_APPROVED,
                             params={
-                                'node': parent_registration._id,
+                                'node': parent_registration.registered_from_id,
+                                'registration': parent_registration._id,
```

Comment:
I added it in because the other function that creates an `EMBARGO_APPROVED` log has it, but looking at the log template itself I don't think it's used. I can remove it